app.json

```
@@ -40,10 +45,14 @@
       [
         "expo-splash-screen",
         {
-          "image": "./assets/images/splash-icon.png",
+          "image": "./assets/images/icons/splash-icon-light.png",
```

splash icons are in "./assets/images/", so this should be changed?
```suggestion
          "image": "./assets/images/splash-icon-light.png",
```